0066-swiftui-and-state-management-pt2/StateManagement.playground/Contents.swift

```diff
@@ -89,22 +89,21 @@ private func ordinal(_ n: Int) -> String {
 
 import Combine
 
-class AppState: BindableObject {
-  var count = 0 {
-    willSet { self.willChange.send() }
-  }
+class AppState: ObservableObject {
+  @Published var count = 0
+  @Published var favoritePrimes: [Int] = []
+}
 
-  var favoritePrimes: [Int] = [] {
-    willSet { self.willChange.send() }
-  }
+struct PrimeAlert: Identifiable {
+  let prime: Int
```

Comment:
it seems maybe `Int` is no longer `Identifiable` because I get errors when using a `Binding<Int>` with the `alert(item:content:)` method. So, I'm wrapping the alert prime in this struct so that I can make it `Identifiable`,

Reply:
Interesting...